### PR TITLE
add release versioning instructions

### PR DESCRIPTION
**Pull request by JimBugwadia**
Signed-off-by: Jim Bugwadia <jim@nirmata.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,3 +32,28 @@ Active voice is preferred in most writing examples. Ex., "this ClusterPolicy mut
...
 * Anything intended to be proper code or typed at a CLI is formatting using Markdown code syntax with backticks or in blocks (surrounded by three backticks).
 * Code represented in blocks should prefer a syntax declaration for this theme's highlighting ability. Ex., when displaying YAML notate the code block with three backticks and "yaml".
+
+## Managing Release Versions
+
+Here are the rules for managing release versions:
+
+1. All fixes and feature changes go to the `main` (we may in a few rare cases make fixes to prior versions of the documentation.) The main branch can be accessed at `https://main.kyverno.io`.
+
+2. When a new release is ready for GA, a new release branch is created (see steps below). Release branches are named `release-{major}-{minor}-{patch}` for example `release-1-4-2`. The release branch can be accessed using the `{branch}.nirmata.io` and the latest release is available at `kyverno.io`.
```
Comment (chipzoller) [inline README.md:42]: It'd be `{branch}.kyverno.io` right? :)

File: README.md [hunk 2]
```diff
@@ -6,7 +6,8 @@ https://kyverno.io
 
 * This site makes use of the [Docsy](https://docsy.dev) theme.
   [Hugo Extended](https://gohugo.io/getting-started/installing#fetch-from-github) is required to render it.
-* `git clone https://github.com/kyverno/website kyverno-website/ --recurse-submodules`
+* Create a fork of the Kyverno website repository to your GitHub account.
+* `git clone https://github.com/{GITHUB-ID}/website kyverno-website/ --recurse-submodules`
```
Comment (chipzoller) [inline README.md:10]: The `clone` would still have to reference `kyverno/website`:  `git clone https://github.com/kyverno/website kyverno-website/ --recurse-submodules`

File: README.md [hunk 3]
```diff
@@ -31,3 +32,28 @@ Active voice is preferred in most writing examples. Ex., "this ClusterPolicy mut
...
+
+1. Create and push the branch using `git checkout -b release-{major}-{minor}-{patch}` or via [GitHub](https://github.com/kyverno/website/branches).
+
+2. [Update Netlify](https://app.netlify.com/sites/kyverno/settings/deploys#branches) to point `production` to the new release.
+
+In the `main` branch:
+
+1. Update the versions list in [config.toml](/config/_default/config.toml) to add the next release.
+
+2. Update `version_menu` and `version` in [params].toml(/config/_default/config.toml) for the next release.
```
Comment (chipzoller) [inline README.md:56]: needs to link to `params.toml`; fix link

**Review by chipzoller — CHANGES_REQUESTED** [review]
We should probably try and add instructions, either in this PR or another, which outlines a process for submitting PRs that need to apply to multiple branches. This could be in the case of updates to rendered policy Markdown files, addressing typos/bugs across multiple releases, etc.

File: README.md (hunk 1 above)
Comment (JimBugwadia) [reply README.md:42]: No, the latest release (e.g. currently 1.4.1) should be on https://kyverno.io as we update the Netlify setting.

File: README.md (hunk 2 above)
Comment (JimBugwadia) [reply README.md:10]: No, ideally you would want to clone your fork of `kyverno/website`. Not all maintainers have permissions to create additional branches in the `kyverno/website` repo. So the recommended process is a [fork & pull](https://movi.hashnode.dev/how-to-successfully-fork-clone-signoff-and-make-a-pull-request-ckdyt03sy06utjas18lx1cjer).

File: README.md (hunk 1 above)
Comment (chipzoller) [reply README.md:42]: I know, but you put `nirmata.io` and not `kyverno.io`.

File: README.md (hunk 2 above)
Comment (chipzoller) [reply README.md:10]: Right, ok, so these are two separate instructions. Step one is create a fork (let's make it easy and link them to https://docs.github.com/en/get-started/quickstart/fork-a-repo). Step two is `git clone`. I'd mention that the `website` bit is by default but if a user changed this to a custom value then it should be `{REPO-NAME}`.

File: README.md [hunk 4]
```diff
@@ -31,3 +32,36 @@ Active voice is preferred in most writing examples. Ex., "this ClusterPolicy mut
...
+
+In the `main` branch:
+
+1. Update the versions list in [config.toml](/config/_default/config.toml) to add the next release.
+
+2. Update `version_menu` and `version` in [params.toml](/config/_default/params.toml) for the next release.
+
+3. Create a PR.
+
+#### Submitting a PR to multiple release branches
```
Comment (chipzoller) [inline README.md:60]: Thank you, this will help those (such as myself) for whom this is advanced git.

**Review by chipzoller — CHANGES_REQUESTED** [review]
Couple things and then done.

File: README.md (hunk 1 above)
Comment (JimBugwadia) [reply README.md:42]: Doh! Will fix.

File: README.md (hunk 2 above)
Comment (JimBugwadia) [reply README.md:10]: Ok, the fork instruction is at the top of the page, but I can restate here for completeness.

Comment (JimBugwadia) [reply README.md]: restructured this section, for clarity

**Comment by JimBugwadia** [comment]
Preview at: https://github.com/kyverno/website/blob/70be2bb3e6cc02c8381cc6ab1f963d1caf7d7760/README.md

**Review by chipzoller — APPROVED** [review]

**Pull request merged**